.travis.yml

```diff
@@ -13,6 +13,8 @@ matrix:
           script: source $TRAVIS_BUILD_DIR/.travis/build_doc.sh
         - python: '3.6'
           env: UPLOAD_COVERAGE=true
+        - python: '3.7'
+          env: UPLOAD_COVERAGE=true
```

Comment:
We don't need UPLOAD_COVERAGE twice (it is already checked in 3.6).
But python 3.7 needs sudo, so change the env line for this:
`sudo: true`

For more on this see: https://github.com/travis-ci/travis-ci/issues/9815